code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -649,21 +645,26 @@
 	unaffected_species = IS_MACHINE
 	var/last_taste_time = -10000
 
-/datum/reagent/hyronalin/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+/decl/reagent/hyronalin/initialize_data(newdata)
+	var/list/data = newdata
+	LAZYSET(data, "last_taste_time", 0)
+	return data
```

Comment:
```suggestion
	LAZYSET(newdata, "last_taste_time", 0)
	return newdata
```

We're not doing a Copy() so this is equivalent I think. No need to define a new var/list.